### ISSUE TITLE
Feature request: hide intro skip button after intro duration

### ISSUE THREAD

**Issue by UriSheinman** [jumoog/intro-skipper#34]
Some intros are 50 seconds and some are 10, and the button should appear for the duration of the intro. Is that possible?

**Comment by jumoog** [comment]
No, it is not possible. The plugin has no control over the button.

**Comment by UriSheinman** [comment]
> No, it is not possible. The plugin has no control over the button.

It obviously does since this is a setting you can change
![Screenshot_20240301-141158](https://github.com/jumoog/intro-skipper/assets/132785427/6ca78b78-30f4-4d25-88b4-731c3a30357d)
Hide skip prompt after, it should have an option to be the time of each intro.

**Comment by jumoog** [comment]
Which app do you have the problem with? The settings are for the website button.

**Comment by UriSheinman** [comment]
> Which app do you have the problem with? The settings are for the website button.

I use findroid, but it gets it's data directly from the plugin. This is a plugin side issue not a cliebt side issue.

**Comment by UriSheinman** [comment]
I dont think the problem is fully understood. Currently you can set the button to hide after a set time (no matter the show). So for a show that has a 2 minute intro and for a show that has a 10 seconds one, the button will show for the same duration no matter what. What should happen is that the time the intro skip button appears for isthe same as the duration of the intro.

**Comment by jumoog** [comment]
findroid uses showSkipPromptAt and hideSkipPromptAt. 
For your change request, they would have to add an option and display the button as long as the value in IntroEnd.
I can change that without breaking anything. Here an example of the Data:
```JSON
{
    "EpisodeId": "e7f5b4d4febd94eee8359aa9e92f8a25",
    "Valid": true,
    "IntroStart": 0,
    "IntroEnd": 88.408,
    "ShowSkipPromptAt": 0,
    "HideSkipPromptAt": 10
}
```

**Comment by UriSheinman** [comment]
Is it not possible to do that on the plugin side? Provie the show at and hide at as variable of intro length?

**Comment by jumoog** [comment]
#45

**Comment by AbandonedCart** [comment]
Wow, I don't think that timing could have lined up any better. I just didn't want to manually edit the xml anymore lol

**Comment by jumoog** [comment]
okay its live in v0.1.10.0. Thank you @AbandonedCart

**Comment by AbandonedCart** [comment]
This also resolved https://github.com/ConfusedPolarBear/intro-skipper/issues/188

**Comment by UriSheinman** [comment]
> This also resolved [ConfusedPolarBear#188](https://github.com/ConfusedPolarBear/intro-skipper/issues/188)

Almost like this was an issue that has bugged me in forever...

**Comment by AbandonedCart** [comment]
Since there is no commenting on issues or submitting PRs to an archived repository, the least that could be done is tagging it for posterity.
